src/vs/workbench/api/common/configurationExtensionPoint.ts

```diff
@@ -78,6 +78,13 @@ const configurationEntrySchema: IJSONSchema = {
 								},
 								description: nls.localize('scope.markdownEnumDescriptions', 'Descriptions for enum values in the markdown format.')
 							},
+							enumItemLabels: {
+								type: 'array',
+								items: {
+									type: 'string'
+								},
+								description: nls.localize('scope.enumItemLabels', 'Labels for enum values. When specified, the {0} values still show after the labels, but less prominently.', '`enum`')
```

Comment:
You might want to mention that they display in the settings UI, since I don't think they appear in the suggest widget in a settings.json

Reply:
Also, if you are using markdown backticks on `enum`, shouldn't this be markdownDescription?